influxdb-mixin/README.md

```diff
@@ -0,0 +1,128 @@
+# InfluxDB mixin
+
+The InfluxDB mixin is a set of configurable Grafana dashboards and alerts.
+
+The InfluxDB mixin contains the following dashboards:
+
+- InfluxDB cluster overview
+- InfluxDB instance overview
+- InfluxDB logs overview
+
+and the following alerts:
+
+- InfluxDBWarningTaskSchedulerHighFailureRate
+- InfluxDBCriticalTaskSchedulerHighFailureRate
+- InfluxDBHighBusyWorkerPercentage
+- InfluxDBHighHeapMemoryUsage
+- InfluxDBHighAverageAPIRequestLatency
+- InfluxDBSlowAverageIQLExecutionTime
+
+## InfluxDB cluster overview
+
+The InfluxDB cluster overview dashboard provides details on the cluster's performance and highlights top instances. The dashboard covers all available aspects of InfluxDB performance and integration health, including Golang performance, query/request load, and task scheduler activity.
+
+![First screenshot of the InfluxDB cluster overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/influxdb/screenshots/influxdb_cluster_overview_1.png)
+![Second screenshot of the InfluxDB cluster overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/influxdb/screenshots/influxdb_cluster_overview_2.png)
+![Third screenshot of the InfluxDB cluster overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/influxdb/screenshots/influxdb_cluster_overview_3.png)
```

Comment:
Screenshots haven't been uploaded yet, are the ones in the PR description the ones you'd like us to upload?

Reply:
I was planning to wait until after the mixin was approved to avoid having to re-upload screenshots (eg. after the go panels are potentially moved), but the ones in the PR description are currently the most accurate.

Reply:
All good, let's just upload the finished ones when we have them then